app/src/main/res/layout/fragment_events.xml

```diff
@@ -87,7 +87,12 @@
                     <FrameLayout
                         android:layout_width="match_parent"
                         android:layout_height="match_parent">
-
+                        <TextView
+                            android:id="@+id/noEventsMessage"
+                            android:layout_width="match_parent"
+                            android:layout_height="@dimen/empty_view_height"
```

Comment:
Make it wrap content

Reply:
@iamareebjamal please review again, I've made my change, I was having problem to make the empty-event message in the center of screen, but then I figured out that the NestedScrollView doesn't fill up the screen when there are no enough data. So I put the fillViewPort="true" in.